common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -216,6 +225,12 @@ class @Problem
         flag = false
     return flag
 
+  # Review button click brings user back to top of problem
+  review_question_click: =>
+    $('html, body').animate({
+      scrollTop: @questionTitle.offset().top
+    }, 500);
+    @questionTitle.attr('tabIndex', '-1').focus();
```

Comment:
In the next story, I believe that focus will go to the problem meta section (a place that doesn't quite exist right now). So I'm OK with this for now, but I think ultimately we should put the tabIndex value in the HTML originally instead of adding it dynamically.